contracts/BeakerContract.sol

```diff
@@ -236,12 +281,59 @@ contract BeakerContract is IKernel {
                 err := add(77,mul(100,mload(retLoc)))
             }
             mstore(0xd, err)
-            revert(0xd,retSize)
+            return(0xd,retSize)
         }
     }
     return err;
```

Comment:
I haven't mentioned this, but `reverts` and `returns` make the the internal solidity returns unreachable. The tests still pass in this case, so we can fix this later.

Reply:
Yep, I did understand that simply considered that it was the desired behaviour. There is not specific reason otherwise.

Reply:
The reason is for handling an `error` value when the syscall returns on the procedure-side, which is unreachable right now.